scss/_utilities.scss

```diff
@@ -358,8 +358,19 @@ $utilities: map-merge(
       values: $spacers
     ),
     // Text
+    "font-size": (
+      rfs: true,
+      property: font-size,
+      class: fs,
+      values: $font-sizes
+    ),
+    "font-style": (
+      property: font-style,
+      class: f,
```

Comment:
I don't think `f-normal` is clear enough, maybe use the `fst` prefix (or something else?)

Reply:
Yeah, playing with this for sure still. Tailwind does `.italic`/`.no-italic`, the latter of which makes me laugh a little bit haha. Wanted to keep it somewhat familiar and `.fst` does fit nice enough. Any other ideas?

Reply:
Not atm. `no-italic` feels very "unlike any other class".